scipy/optimize/tests/test_linesearch.py

```diff
@@ -17,8 +17,8 @@ def assert_wolfe(s, phi, derphi, c1=1e-4, c2=0.9, err_msg=""):
     phi0 = phi(0)
     derphi0 = derphi(0)
     derphi1 = derphi(s)
-    msg = "s = {}; phi(0) = {}; phi(s) = {}; phi'(0) = {}; phi'(s) = {}; {}".format(
-        s, phi0, phi1, derphi0, derphi1, err_msg)
```

Comment:
This change to fix this CI error which is not related the bug: https://github.com/scipy/scipy/actions/runs/8467991554/job/23199926867